test/integration/node-specific/resource_tracking_script_builder.ts

```diff
@@ -27,7 +26,6 @@ export type ProcessResourceTestFunction = (options: {
   uri?: string;
   log?: (out: any) => void;
   expect: typeof expect;
-  mongodb?: typeof mongodb;
```

Comment:
All tests that use this property are skipped now (all of them inside `test/integration/node-specific/client_close.test.ts`), that's why commenting out this property is not breaking tests. At the same time this is not a proper solution and those tests (or this setup) needs to be updated.

Maybe we can do something similar to the dependency test, i.e. run `ts-node`, provide `cwd` and import entities directly from `src/` folder. At the same time I don't know why those tests are skipped, maybe they are not relevant anymore.